libraries/Microsoft.Bot.StreamingExtensions/Payloads/IRequestManager.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.StreamingExtensions.Payloads
+{
+    internal interface IRequestManager
+    {
```

Comment:
Why is an interface needed? There's only 1 implementation. Is this just for mocking?

Reply:
just for mocking